### ISSUE TITLE
Highlighting own username not working.

### ISSUE THREAD

**Issue by Ellron** [night/BetterTTV#315]
I first noticed that my own username wasn't being highlighted March 29, 2015.
My twitch username is "Ellron23" and I've had the following set as highlight words for quite some time: "Ellron23 (Ellron23) ellron". I like to be able to see my own messages differentiated from the rest of the Kappa spam. I asked a friend to test out highlighting his own message on his system, and he was getting the same issue (own message not highlighting).

In conclusion:
Expected result: Own message highlights when own name is set as a highlight key phrase.
Observed result: Own messages are not highlighted when own name is set as highlight key phrase.

**Comment by vendethiel** [comment]
@night Isn't that the same argument as the other issue with "our own messages aren't highlighted"?

**Comment by Ellron** [comment]
Sorry if it is a duplicate, I wasn't able to figure out how to view all the bug reports to check.

**Comment by night** [comment]
Vendie broke it, vendie fixes.

**Comment by Ellron** [comment]
Thanks so much guys!
